.deploy/inspect.sh

```
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+get_logs_by_version() {
+    local SERVICE_NAME=$1
+    
+    # Get current version number
+    CURRENT_VERSION=$(docker service inspect --format '{{.Version.Index}}' $SERVICE_NAME)
```

Shellcheck (0.10.0)

[warning] CURRENT_VERSION appears unused. Verify use (or export if used externally).

(SC2034)